### PR TITLE
test: Add system integration test for MSSQL row validation with different datatypes

### LINKED ISSUE #657: MSSQL (add different data types for row hash in system Integration test)
This is sub-issue for issue#611. This should involve adding different data types test in the row validation (string/varchar, integer, decimal, & date)

### PR DESCRIPTION
**Pull request by sharangagarwal**
Closes #657 

Adds system integration test for MSSQL row validation with different datatypes

- Added file mssql_data_row.sql to generate dummy data with serial, int, text, char, varchar, float, decimal, datetime and date datatypes
- Added test_mssql_row function in tests/system/data_sources/test_mysql.py to perform row hash validation for different datatypes

### CODE REVIEW

File: tests/system/data_sources/deploy_cloudsql/mssql_data_row.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+DROP TABLE IF EXISTS test_data_types_mysql_row;
+CREATE TABLE test_data_types_mysql_row(
+	serial_col int AUTO_INCREMENT ,
+    int_col int(3),
+	text_col  char(100),
```
Comment (Raniksingh) [inline tests/system/data_sources/deploy_cloudsql/mssql_data_row.sql:19]: Can we align all of these statements?

Comment (sharangagarwal) [reply tests/system/data_sources/deploy_cloudsql/mssql_data_row.sql:19]: aligned the whitespaces

**Review by Raniksingh — APPROVED** [review]

**Pull request merged**